### PR TITLE
Task 3. Create min.py script

### PR DESCRIPTION
**Pull request by AnnaLangner**

### CODE REVIEW

**Review by klangner — CHANGES_REQUESTED** [review]
The parameters should be provided during program run as command line parameters

File: script/min.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+x = input('Entry first number:')
+y = input('Entry second number:')
+if float(x) > float(y) :
```
Comment (klangner) [inline script/min.py]: What about situation when x == y? Something should be printed.

**Review by klangner — CHANGES_REQUESTED** [review]

**Comment by AnnaLangner** [comment]
changes have been made

File: script/min.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#This script should take 2 parameters (numbers) and print lower number on the console.
+import argparse
+parser = argparse.ArgumentParser()
```
Comment (klangner) [inline script/min.py:5]: Please add 2 empty lines after imports.

File: script/min.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+parser.add_argument("num1", help="num is the int you use here")
+parser.add_argument("num2", help="num is the int you use here")
+args = parser.parse_args()
+if float(args.num1) > float(args.num2) :
```
Comment (klangner) [inline script/min.py]: Please remove space before ':' (in all lines)

File: script/min.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+parser = argparse.ArgumentParser()
+parser.add_argument("num1", help="num is the int you use here")
+parser.add_argument("num2", help="num is the int you use here")
+args = parser.parse_args()
+if float(args.num1) > float(args.num2) : 
+  print("Lower number:", args.num2)
+elif float(args.num1) < float(args.num2) :
+  print("Lower number:", args.num1)
+else :
+  print("Numbers are equal")
```
Comment (klangner) [inline script/min.py]: Pleas add one empty line at the end of the file

**Review by klangner — CHANGES_REQUESTED** [review]

**Comment by AnnaLangner** [comment]
changes have been made

**Comment by klangner** [comment]
Ok. Good job.

**Pull request merged**